jbig2/jbig2_decompress_image.go

```diff
@@ -0,0 +1,57 @@
+/*
+ * This example showcases the decompression of the jbig2 encoded image and storing into
+ * commonly used jpg format.
+ *
+ * As the input for this test the result of the compression example would be used (lossless image).
+ * The result of this example is an image that has unchanged quality (in compare to the input of the compression example).
+ */
+
+package main
+
+import (
+	"image/jpeg"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/unidoc/unipdf/v3/core"
+)
+
+func main() {
+	// read jbig2 encoded file with the checkerboard-squares-black-white image.
```

Comment:
Pretty obvious, maybe comment not required?

Reply:
Good idea. Done.